res/generate_update_metadata.py

```
@@ -0,0 +1,277 @@
+    check_version.set_defaults(func=command_check_version)
+    return parser
+def main(argv=None):
+    parser = build_parser()
+    args = parser.parse_args(argv)
+    args.func(args)
+if __name__ == "__main__":
+    try:
+        main()
+    except SystemExit:
+        raise
+    except Exception as error:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)